CONTRIBUTING.md

```diff
@@ -1,7 +1,56 @@
 # Contributing
 
+## Overview
 This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
 
 When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repositories using our CLA.
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Testing
+We have three types of testing:
```

Comment:
Do we want to also mention coveralls and acceptance criteria for tests? I do like the breakdown between Unit using go test, then ginkgo for behavioral testing.

Reply:
Probably worth mentioning coveralls. What do you mean by acceptance criteria?

Reply:
i was thinking code coverage for all new functions, but maybe coveralls does enough